libnavigation-base/src/main/java/com/mapbox/navigation/base/options/NavigationOptions.kt

```diff
@@ -64,7 +64,7 @@ data class NavigationOptions(
         private var timeFormatType: Int = TimeFormat.NONE_SPECIFIED
         private var navigatorPredictionMillis: Long = DEFAULT_NAVIGATOR_PREDICTION_MILLIS
         private var distanceFormatter: DistanceFormatter? = null
-        private var onboardRouterOptions: OnboardRouterOptions? = null
+        private var onboardRouterOptions: OnboardRouterOptions? = OnboardRouterOptions.Builder().build()
```

Comment:
Can this be non-null then? 🤔 This also applies to the `Builder` / `onboardRouterOptions`. Wondering also if we could make `OnboardRouterOptions` from `NavigationOptions` non-null as well 👀 https://github.com/mapbox/mapbox-navigation-android/blob/e93571b162c83eadfef56c01ef0c1961b6cd22fc/libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt#L615-L616 https://github.com/mapbox/mapbox-navigation-android/blob/e93571b162c83eadfef56c01ef0c1961b6cd22fc/libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt#L69-L71